### PR TITLE
Use discoverActivityOptions for refresh schema activity

### PR DESCRIPTION
**Pull request by alovew**
Use DiscoveryActivityOptions for Refresh Schema Activity to increase scheduleToCloseTimeout to 2 hours

### CODE REVIEW

**Review by benmoriceau — APPROVED** [review]

**Review by malikdiarra — APPROVED** [review]

File: airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java [hunk 1]
```diff
@@ -88,7 +90,9 @@ public StandardSyncOutput run(final JobRunConfig jobRunConfig,
         Workflow.getVersion(AUTO_DETECT_SCHEMA_TAG, Workflow.DEFAULT_VERSION,
             AUTO_DETECT_SCHEMA_VERSION);
 
-    if (autoDetectSchemaVersion >= AUTO_DETECT_SCHEMA_VERSION) {
+    final int discoveryOptionsVersion = Workflow.getVersion(DISCOVERY_OPTIONS_TAG, Workflow.DEFAULT_VERSION, DISCOVER_OPTIONS_VERSION);
```
Comment (alovew) [inline airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java:93]: @benmoriceau I added a new version here, but still seeing test failures

Comment (benmoriceau) [reply airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java:93]: I think that we should have 2 `private RefreshSchemaActivity refreshSchemaActivity;` with the different options. Then you'll need to use the new version to call one or the other activity.

Comment (alovew) [reply airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java:93]: Just tried this, but I'm still seeing that test failure

Comment (alovew) [reply airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java:95]: just pushed up my changes - is that what you meant?

Comment (benmoriceau) [reply airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java:95]: The issue is in the stack trace of the failing test, The bean is not properly defined, I'm trying to make a fix.

Comment (benmoriceau) [reply airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java:95]: I made a fix for the workflow replayer, another test need to be updated.

**Pull request merged**